rest/src/test/java/com/redhat/runtimes/inventory/web/DisplayInventoryTest.java

```diff
@@ -0,0 +1,70 @@
+/* Copyright (C) Red Hat 2023 */
+package com.redhat.runtimes.inventory.web;
+
+import static com.redhat.runtimes.inventory.models.Constants.X_RH_IDENTITY_HEADER;
+import static io.restassured.RestAssured.given;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import io.quarkus.test.common.QuarkusTestResource;
+import io.quarkus.test.junit.QuarkusTest;
+import io.restassured.http.Header;
+import io.vertx.core.json.JsonObject;
```

Comment:
We're already using Jackson - why are we bringing in an additional JSON handling library?

Reply:
Good point, I'll switch this over to jackson